### ISSUE TITLE
Pacman completion broken

### ISSUE THREAD

**Issue by flying-sheep** [rsteube/carapace-bin#1294]
```console
$ carapace pacman nushell pacman -R fil | jaq
[
    {"value": "filERR", "description": "unknown shorthand flag: 'R' in -R"},
    {"value": "fil_"}
]
```

Similar for other commands

**Comment by rsteube** [comment]
I'm in the process of switching to an own pflag fork: https://github.com/rsteube/carapace-pflag to better support non-posix variants.
Seems to be fixed there already.

needs #1293
needs https://github.com/rsteube/carapace/pull/560

**Comment by rsteube** [comment]
@flying-sheep  Is this just related to `pacman`? Seems to work fine in `master` branch.

**Comment by flying-sheep** [comment]
Oops, seems like it! Sorry for not trying master first

**Comment by rsteube** [comment]
No, the ticket is right - I missed that. Was just checking if there is something else before i tag a release.

**Comment by flying-sheep** [comment]
I was only trying with that one, and can’t think of any other commands that have flag syntax for subcommands :thinking:

**Comment by rsteube** [comment]
fixed in v0.15.1
